src/AzureSqlActionHelper.ts

```diff
@@ -116,28 +125,39 @@ export default class AzureSqlActionHelper {
     }
 
     /**
-     * SqlPackage.exe can be installed in two ways:
-     *  1. SQL Server Management Studio (SSMS) and the Dac Framework MSI installs it in location C:/Program Files/Microsoft SQL Server/{SqlVersopn}/DAC/bin/SqlPackage.exe'
-     *  2. SSDT (SQL Server Data Tools) installs it in location VS Install Directory/Common7/IDE/Extensions/Microsoft/SQLDB/DAC/{SqlVersion}
+     * SqlPackage.exe can be installed in four ways:
+     *  0. as a global dotnet tool
+     *  1. SQL Server Management Studio (SSMS) used to install it to in location C:/Program Files/Microsoft SQL Server/{SqlVersopn}/DAC/bin/SqlPackage.exe' (REMOVE in the future)
```

Comment:
How was this order determined? dotnet tool as `0` makes a ton of sense to me. Curious why SSMS is next on the pecking order.

Reply:
I'm not sure, to be honest - this was existing code and is consistent with the ADO task.
I checked current SSMS installs and it doesn't bring SqlPackage.exe, so this shouldn't ever get picked up.  I left a note to remove this in the future (that should be a work item but I'm lazy)